db/migrate/20180412074903_change_amount_field_type.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeAmountFieldType < ActiveRecord::Migration[5.1]
```

Comment:
You can roll back and change the first migration until it came to production(or merged).